internals/delivery/grpc/handler.go

```diff
@@ -0,0 +1,47 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+	"github.com/chernyshevvladislav/test-grpc/.bin/proto/library"
+	"github.com/chernyshevvladislav/test-grpc/internals/service"
+)
+
+type Handler struct {
+	Services Services
+	library.UnimplementedLibraryServer
+}
+
+type Services struct {
+	BooksService   service.Book
+	AuthorsService service.Author
+}
+
+func NewHandler(bookService service.Book, authorService service.Author) *Handler {
+	return &Handler{
+		Services: Services{
+			BooksService:   bookService,
+			AuthorsService: authorService,
+		},
+	}
+}
+
+func (h *Handler) GetAuthorsByBookTitle(ctx context.Context, req *library.GetAuthorsByBookTitleRequest) (*library.GetAuthorsByBookTitleResponse, error) {
+	resp, err := h.Services.AuthorsService.FindAuthorsByBookTitle(ctx, req.Title)
+	if err != nil {
+		_ = fmt.Errorf("cannot find Authors: %s", err)
```

Comment:
А это чё за херня?
Должно быть так:
```suggestion
		return nil, fmt.Errorf("cannot find Authors: %w", err)
```

Reply:
Почитай про обработку ошибок в Golang и что делает %w в fmt.Errorf